src/sentry/relay/globalconfig.py

```diff
@@ -0,0 +1,5 @@
+def get_global_config():
```

Comment:
Can we instrument this with `@metrics.wraps` so we get a timing for this? You don't need the explicit `incr` anymore after that.